server/src/types/inputs/ResourceRequestInput.ts

```diff
@@ -53,9 +55,18 @@ export class ResourceRequestInput {
   @Field({ nullable: true })
   comments?: string;
 
+  @ValidateIf(
+    (o) =>
+      o.status === "Outsourced" ||
+      o.status === "Moved" ||
+      o.status === "Hired" ||
+      o.status === "Over allocated"
```

Comment:
use array and check it includes the status

Reply:
includes doesn't work